### ISSUE TITLE
v0.1.3: unable to resolve symbol forcat

### ISSUE THREAD

**Issue by timgilbert** [webnf/lein-collisions#2]
Hi! I'm getting this exception running `lein collisions` with `:plugins [... [lein-collisions "0.1.3"]]` in my project.clj:

```
clojure.lang.Compiler$CompilerException: java.lang.RuntimeException: Unable to resolve symbol: forcat in this context, compiling:(leiningen/collisions.clj:27:4)
 at clojure.lang.Compiler.analyze (Compiler.java:6464)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler$InvokeExpr.parse (Compiler.java:3665)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6646)
    ...
```

I couldn't find any docs on a `(forcat)` function in the clojure libs, maybe you've got a dependency missing or something?

**Comment by bendlas** [comment]
I messed up the dependency repackaging, sorry. Fixed with `0.1.4`

**Comment by timgilbert** [comment]
Thanks! `0.1.4` works for me.
